database/abuseemail.go

```diff
@@ -151,6 +154,16 @@ func (a AbuseEmail) result() ([]string, []string) {
 	return blocked, unblocked
 }
 
+// Sender is a helper function that returns the sender of the email, by default
+// it returns the field from the ReplyTo header but it falls back to the From
+// field if that was empty
+func (a AbuseEmail) Sender() string {
```

Comment:
I think it would bit a tiny bit more accurate to call this `ReplyTo`.

Reply:
yeah I originally made it `Sender` because I already have a `ReplyTo` field that contains the value from the `ReplyTo` header, I updated it to `ReplyToEmail` to avoid the naming conflict